src/components/Button/Button.css

```diff
@@ -11,15 +11,17 @@ body {
   height: 100%; }
 
 .Button button {
-  background-color: #3ee7ad;
-  border: none;
   font-family: 'Assistant';
   font-size: 20px;
   padding: 10px 0px;
   color: #f7f8f9;
   border-radius: 3px;
   cursor: pointer; }
 
+.Button button {
+  background-color: #3ee7ad;
+  border: none; }
```

Comment:
Why did you separate these into two css objects instead of the one?

Reply:
Doh!